src/sentry/api/endpoints/project_rules_configuration.py

```diff
@@ -15,9 +17,13 @@ def get(self, request, project):
         condition_list = []
         filter_list = []
 
+        project_has_filters = features.has("projects:alert-filters", project)
         # TODO: conditions need to be based on actions
         for rule_type, rule_cls in rules:
             node = rule_cls(project)
+            # skip over conditions if they are not in the migrated set for a project with alert-filters
+            if project_has_filters and node.id not in SENTRY_RULES_WITH_MIGRATED_FILTERS:
```

Comment:
I thought about that too, but I think it's alright to provide extra filters. Previously every filter we added would be served to the frontend, but because of the feature flagging it would not be displayed so users don't have access to them anyway when creating a rule. Even if a user chose to use our api and manually make a rule with filters it would also be fine since we would want them to use filters in the future anyway. 

Also we already enabled filters for sentry and some rules are using them. However, we haven't "officially" migrated the sentry rules so getting rid of the filters might give us some trouble.